docs/usage/types.md

```diff
@@ -85,9 +85,17 @@ with custom properties and validation.
 `typing.Tuple`
 : see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
 
+`subclass of typing.NamedTuple (or collections.namedtuple)`
+: Same as `tuple` but instantiates with the given namedtuple.  
+ _pydantic_ will validate the tuple if you use `typing.NamedTuple` since fields are annotated.  
+  If you use `collections.namedtuple`, no validation will be done.
+
 `typing.Dict`
 : see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
 
+`subclass of typing.TypedDict`
```

Comment:
```suggestion
`typing.TypedDict`
```
I think the subclass is implicit

Reply:
It used to be the case for me too but I now tend to disagree since we added the support of `enum.Enum` to have a valid enum instance. We could someday add `typing.TypedDict` just to have a valid `TypedDict` instance for example. WDYT?

Reply:
I would still prefer to remove `subclass of` and instead be explicity when we mean the actual type, like `Enum`.